tutorials/s4sdk-caching/s4sdk-caching.md

```diff
@@ -51,268 +56,136 @@ If each of these options apply to your use case, then we highly recommend the ca
+In [the resilience tutorial] (https://developers.sap.com/tutorials/s4sdk-resilience.html), we introduced resilience
+into our application, using `Resilience4j` library. Now, in order to make our OData calls cacheable, we will extend
+the use of `ResilienceConfiguration` class used there and configure caching as well.
 
-1. Provide a constructor. Here we simply add the default constructor.
-2. Override the `runCacheable`() method. As you might have noticed already, we can simply take the code we used to call our OData service via the Virtual Data Model from the previous blog posts, and put it into the method. You can even specify Java exceptions. So no changes needed!
-3. Override the `getCache`() method. Since it might be possible for you to use any cache utility, you need to instantiate the Cache object yourself and provide a method for the internal usage.
+Add the following lines at the end of the constructor of the `GetBusinessPartnerCommand`:
 
-Now that we have a working command with enabled cache features, we can adapt our `BusinessPartnerServlet`.
+`./application/src/main/java/com/sap/cloud/sdk/tutorial/GetBusinessPartnersCommand.java`
 
-**Note:** With regard to the `Hystrix` example of the previous blog post, the only change will be a different class name of the command.
+```java
+final ResilienceConfiguration.CacheConfiguration cacheConfig =
+        ResilienceConfiguration.CacheConfiguration
+                .of(Duration.ofSeconds(10))
+                .withoutParameters();
 
+myResilienceConfig.cacheConfiguration(cacheConfig);                         
 ```
-package com.sap.cloud.sdk.tutorial;
-
-import com.google.gson.Gson;
-import org.slf4j.Logger;
 
-import javax.servlet.ServletException;
-import javax.servlet.annotation.WebServlet;
-import javax.servlet.http.HttpServlet;
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
-import java.io.IOException;
-import java.util.List;
+As mentioned above, we use our `ResilienceConfiguration` to integrate the caching functionality, which is described in a `CacheConfiguration`. There are two steps to our configuration:
 
-import com.sap.cloud.sdk.cloudplatform.logging.CloudLoggerFactory;
+1. Determine how long objects are to be cached
+2. Declare the parameters that need to be stored together with the cached data
 
-import com.sap.cloud.sdk.s4hana.datamodel.odata.namespaces.businesspartner.BusinessPartner;
+The first step is obviously necessary, since we want to set up a cache that is meant to store data only a short amount of time to optimize execution time. How long you want to keep data cached depends on the data that you receive. How fast do you expect the information to be outdated? How frequently will the data be accessed?
```

Comment:
Why does storing a short amount of time optimize execution time? I would rather speak of a trade-off between data being up-to-date and fast response, which depends on the use case at hand.